### PR TITLE
fix(filtermultiselect): fix console warnings

### PR DESCRIPTION
**Pull request by NatalieLoyzaga**
## Why

https://cultureamp.atlassian.net/browse/KDS-694

## What

Add display names to providers/context and update story code block semantics

### CODE REVIEW

File: packages/select/docs/FilterMultiSelect.stories.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Paragraph } from "@kaizen/typography"
 import { FilterMultiSelect, getSelectedOptionLabels } from "@kaizen/select"
 import { Label } from "@kaizen/draft-form"
 import { CodeBlock } from "@kaizen/design-tokens/docs/DocsComponents"
+import { Box } from "@kaizen/component-library/components/Box"
```
Comment (HeartSquared) [inline packages/select/docs/FilterMultiSelect.stories.tsx:10]: If we aren't encouraging usage of Box, we shouldn't use it ourselves either :)

File: packages/select/docs/FilterMultiSelect.stories.tsx [hunk 2]
```diff
@@ -72,10 +73,10 @@ export const DefaultKaizenSiteDemo: ComponentStory<
           </>
         )}
       </FilterMultiSelect>
-      <Paragraph variant={"body"}>
-        Items:{" "}
+      <Box mt={4}>
+        <Paragraph variant={"body"}>Items: </Paragraph>{" "}
```
Comment (HeartSquared) [inline packages/select/docs/FilterMultiSelect.stories.tsx:77]:
```suggestion
        <Paragraph variant="body">Items: </Paragraph>{" "}
```

Nitpick: If we're fixing the line anyway, might as well :)

Comment (NatalieLoyzaga) [reply packages/select/docs/FilterMultiSelect.stories.tsx]: Yep all done :)

**Comment by NatalieLoyzaga** [comment]
@HeartSquared Ready for review :)

File: packages/select/docs/FilterMultiSelect.stories.tsx [hunk 3]
```diff
@@ -72,10 +72,10 @@ export const DefaultKaizenSiteDemo: ComponentStory<
           </>
         )}
       </FilterMultiSelect>
-      <Paragraph variant={"body"}>
-        Items:{" "}
+      <div style={{ marginTop: 4 }}>
+        <Paragraph variant="body">Items: </Paragraph>{" "}
```
Comment (HeartSquared) [inline packages/select/docs/FilterMultiSelect.stories.tsx:76]:
```suggestion
        <Paragraph variant="body">Items: </Paragraph>
```

Doesn't look like you need it (based on the other section) :)

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**